.teamcity/OpenSourceProjects_Storybook/buildTypes/OpenSourceProjects_Storybook_ReactNative.kt

```diff
@@ -30,14 +28,6 @@ object OpenSourceProjects_Storybook_ReactNative : BuildType({
             """.trimIndent()
             dockerImage = "node:%docker.node.version%"
         }
-        script {
-            name = "react-native-vanilla"
-            scriptContent = """
-                cd examples/react-native-vanilla
-                yarn storybook --smoke-test
-            """.trimIndent()
-            dockerImage = "node:%docker.node.version%"
-        }
```

Comment:
There are 2 `--reactnative` flags in this file in other steps. Please correct them.